_blog/2018/egress-tcp.md

```diff
@@ -0,0 +1,306 @@
+---
```

Comment:
Please exclude the .DS_Store binary file that slipped in your commit.

Reply:
Done.